package.json

```diff
@@ -1,33 +0,0 @@
-{
-  "name": "getGitHub",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "test": "node src/test.js",
-    "start": "node src/server.js",
-    "server": "nodemon src/server.js",
```

Comment:
This should be called `watch` or something, since that is what it is doing. `npm start` also starts a server.